app/test_main.py

```diff
@@ -1,3 +1,34 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+@pytest.mark.parametrize("cat_age, dog_age, output", [
+    (0, 0, [0, 0]),
+    (14, 14, [0, 0]),
+    (15, 15, [1, 1]),
+    (27, 27, [2, 2]),
+    (28, 28, [3, 2]),
+    (28, 20, [3, 1]),
+    (100, 100, [21, 17]),
+    (-10, -10, [0, 0])
+])
+def test_animal_age_with_input_data(
+        cat_age: int,
+        dog_age: int,
+        output: int
```

Comment:
You do not expect an integer for `output`, but rather a list of integers. Indicate this in your type annotation.